search-resources-api/src/main/java/no/unit/nva/search2/ExportResourceHandler.java

```diff
@@ -18,15 +18,12 @@
 import no.unit.nva.search2.resource.ResourceSearchQuery;
 import nva.commons.apigateway.ApiS3GatewayHandler;
 import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
 import nva.commons.apigateway.exceptions.BadRequestException;
 import nva.commons.core.JacocoGenerated;
 import software.amazon.awssdk.services.s3.S3Client;
 import software.amazon.awssdk.services.s3.presigner.S3Presigner;
 
-/**
- * @author Sondre Vestad
- * @author Stig Norland
- */
```

Comment:
we have git these days, no need for these.
https://mikemybytes.com/2016/12/29/still-using-author-please-dont/